ci/gpu/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+#########################################
+# cuDF GPU build and test script for CI #
+#########################################
+set -e
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> $@\n"
+}
+
+# Set path and build parallel level
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+
+# Set home to the job's workspace
+export HOME=$WORKSPACE
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Activate conda env..."
+source activate gdf
+
+logger "Check versions..."
+python --version
+$CC --version
+$CXX --version
+conda list
+
+################################################################################
+# BUILD - Build libcudf and cuDF from source
+################################################################################
+CUDF=$WORKSPACE/CUDF-SRC
+
+git clone https://github.com/rapidsai/cudf.git CUDF-SRC && cd $CUDF
+git submodule update --init --remote --recursive
+git checkout branch-0.6
```

Comment:
@mike-wendt I would assume git commands would ideally be pushed to the jenkins job and removed from this script

Reply:
Yeah I'll come back to this